project/assets/newevaluationStyle.less

```diff
@@ -0,0 +1,358 @@
+
+
+body{
+  background-color: rgb(82, 86, 89);
+  margin-top: 3%;
+  margin-left: 0;
+}
+
+
+
+/* ---------------------------------Navigation Bar--------------------------- */
+
+
+nav {
+  display: inline;
+  background-color: whitesmoke;
+  width: 100%;
+}
+
+.navigation-bar {
+  height: 5rem;
+  background-color: whitesmoke;
+  border-bottom: 1px solid darkgrey;
+
+  .navigation-bar-svlogo {
+    width: 10%;
+    height: 50%;
+    margin-left: 5%;
+    margin-top: 0.5%;
+  }
+
+  a {
+    text-decoration: none;
+    font-size: 184%;
+    margin-left: 11%;
+    margin-right: -7%;
+    color: black;
+  }
+
+  .navigation-bar-active {
+    border-bottom: 6px solid DodgerBlue;
+    padding: 1%;
+  }
+
+  .navigation-bar-logout {
+    margin-left: 50%;
+  }
+}
+
+
+
+
+/* -------------------Candidate, interviewer, date inputs---------------------*/
+
+
+.inputs {
+  background-color: whitesmoke;
+  text-decoration: none;
+  padding-top: 5%;
+  padding-left: 21%;
+
+  .inputs-candidate {
```

Comment:
If I'm seeing correctly these `inputs-` classes have the same properties with pretty much the same values. Can they be combined into only one class? And have a class just for the differences?